vnext/ReactUWP/Threading/WorkerMessageQueueThread.cpp

```diff
@@ -88,7 +88,7 @@ struct WorkerMessageQueueThread::Impl {
 };
 
 WorkerMessageQueueThread::Impl::Impl() {
-  HRESULT hr = CreateAsyncWorkQueue(&queue);
+  [[maybe_unused]] HRESULT hr = CreateAsyncWorkQueue(&queue);
```

Comment:
>HRESULT hr [](start = 19, length = 10)

Remove or comment out if it's not used?

Reply:
commented out

---
In reply to: [370871722](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370871722) [](ancestors = 370871722)